### PR TITLE
[Lake] Support export table

### PR DESCRIPTION
**Pull request by TszKitLo40**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Support export of lake table

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 2 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/ExportJob.java|0|2|00.00%|[212, 213]|

**Review by tracymacding — DISMISSED** [review]

**Review by imay — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/load/ExportJob.java [hunk 1]
```diff
@@ -208,6 +208,11 @@ public void setJob(ExportStmt stmt) throws UserException {
         }
 
         this.sql = stmt.toSql();
+
+        // TODO Support snapshot of lake table
+        if (this.exportTable.getType() == Table.TableType.LAKE) {
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/load/ExportJob.java:213]:
```suggestion
        if (exportTable.isLakeTable()) {
```

Comment (TszKitLo40) [reply fe/fe-core/src/main/java/com/starrocks/load/ExportJob.java]: done

**Comment by wyb** [comment]
run starrocks_fe_unittest

**Review by wyb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 2 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/ExportJob.java|0|2|00.00%|[213, 214]|

**Review by imay — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**